pkg/cloud/azure/actuators/machine/reconciler_test.go

```diff
@@ -114,3 +117,132 @@ func TestGetSpotVMOptions(t *testing.T) {
 		})
 	}
 }
+
+func TestSetMachineCloudProviderSpecifics(t *testing.T) {
+	abcZones := []string{"a", "b", "c"}
+
+	testCases := []struct {
+		name                string
+		scope               func(t *testing.T) *actuators.MachineScope
+		vm                  compute.VirtualMachine
+		expectedLabels      map[string]string
+		expectedAnnotations map[string]string
+		expectedSpecLabels  map[string]string
+	}{
+		{
+			name:  "with a blank vm",
+			scope: func(t *testing.T) *actuators.MachineScope { return newFakeScope(t, "worker") },
+			vm:    compute.VirtualMachine{},
+			expectedLabels: map[string]string{
+				"machine.openshift.io/cluster-api-machine-role": "worker",
+				"machine.openshift.io/cluster-api-cluster":      "clusterID",
+			},
+			expectedAnnotations: map[string]string{
```

Comment:
can we use the const we have available for this e.g `MachineInstanceStateAnnotationName`?